### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Rejdesu** [maximmax42/Discord-CustomRP#217] [Ideas]
**Feature:**
Would it be possible to add a drop down menu with different saved (locally) options for the Details and State fields? Possibly even for the other fields?

**Something like this:**
![CustomRP](https://github.com/maximmax42/Discord-CustomRP/assets/29308849/0ad428d9-a215-4e01-a2a6-7cab42068fe5)

I hope I am able to convey my idea thoroughly.

**Comment by maximmax42** [comment]
Hmmm... Interesting idea, though I'm not sure how would you manage it, as in adding and deleting to that list.
